### DISCUSSION TITLE
questions-about-food/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#51] [Announcements]
# questions-about-food/

چند تا سؤال هست که مدتی ذهنم رو درگیر کرده و دارم بهشون فکر می‌کنم.
مسئله‌ام هم مثل مسئله خیلی‌هامون مسئله چاقی و اضافه وزن هست. و اخیراً دارم ویدئوهایی رو می‌بینم که تمام باورهای قبلی‌ام درباره چاقی رو زیرسؤال می‌برند و جالب این که تو سبک غذایی‌ام تغییراتی دادم و «به راحتی» دارم وزن کم می‌کنم.
ولی چون از مطالبی که می‌بینم کاملاً مطمئن نیستم، سؤالاتی که برام پیش میاد و متخصصین امر این موارد رو زیر سوال می‌برند رو اینجا مطرح می‌کنم.

https://aprd.ir/questions-about-food/

**Comment by elhamghadiri** [comment]
سلام .
این مطلب واقعا جالب بود و ذهنم رو حسابی به چالش کشید. با اینکه خودم حدود 4 ساله دارم تو حوزه تغذیه و سبک زندگی سالم و رژیم‌های غذایی یاد میگیرم اما این مطلب کلی نکات جدید داشت که هیچ وقت به ذهنم خطور نکرده بود. 
ممنون که منابع رو معرفی کردید و گذاشتم تو برنامه که برم ببینم و یاد بگیرم. 
من هم اخیرا کتاب "مانایی" رو خوندم و مستند ".Hack your health. The secrets of your gut" رو دیدم که هر دو رو پیشنهاد میکنم .

**Reply by pourmand1376** [reply]
سلام. وقت شما بخیر و خوشی. خوشحالم که مجدداً شروع به نوشتن وبلاگ‌تون می‌کنید. 

بله. من این چند وقت خیلی روی این مطالب حساس شدم و به همین خاطر منابع متعدد و متناقضی رو مطالعه کردم که برام سؤالات بالا ایجاد شدند. 

اتفاقاً از چند وقت پیش که راجع به «میکروبیوم‌های روده» اطلاعاتی پیدا کردم برام جالب بود که بدونم رابطه این‌ها با بدن من چیه و چه تأثیر متقابلی روی هم داریم. خیلی ممنون که معرفی کردید. همین روزها می‌شینم کامل دانلود می‌کنم و می‌بینم.

**Comment by masoudnasrollahi** [comment]
آیا ما راجع‌به سلامتی و بیماری درست فکر می‌کنیم؟
تعریف چاقی چیست؟‌ علت یا علل آن چیست؟‌ آیا چاقی بیماری است؟
آیا هیچ تحقیق نظام‌مندی راجع‌به سلامتی و چاقی وجود داره؟
آیا گرسنگی کشیدن زجر است؟ زجری حاصل از ترک عادت خوردن؟!
آیا میزان اراده داشتن یک فرد را می‌توان معیاری از سلامتی‌اش در نظر گرفت؟
آیا چاقی فقط یک وضعیت بدنی است یا ابعاد دیگری در این وضعیت تاثیرگذارند؟
روند «خوردن» در تاریخ چگونه است؟/تاریخ «خوردن» چیست؟/از آغاز تکامل تا شکم‌های برآمدۀ امروز چگونه مصرف کردیم و خوردیم؟
آیا پزشکی و طب رایج صلاحیت نظر دادن دربارۀ سلامتی به‌عنوان یک وضعیت سیستمی (؟)، به‌مثابۀ یک سیستم پایدار یا کارآمد را داره؟
آیا اجداد خودم را رعیت‌های زمین‌های کشاورزی می‌شمارم یا انسان خوراک‌جوی غارنشین؟
انسان سیستم پیچیده است یا بدن انسان؟ یا هر دو؟ یا هیچ‌کدام؟
آیا هر خوردنی‌ای قوت روزانه است؟
مراحل فرا آوری غذا بر روی میز در حس سیری تاثیری دارد؟
چه قسمت‌هایی از این چرخه تولید و مصرف باعث‌وبانی حال بدمان است؟
مگه جذب باکتری‌ها، قسمت از همۀ جذب بدن نیست؟
آیا مواد غذایی و خوراکی فقط توده‌ای از اجزا مثل پروتئین و کربوهیدرات و گلوکز و فروکتوز و چربی و این چیزا هستند و نسبت و میزان اجزاء معیار تصمیم برای خوردنشونه؟ یا همین اجزاء در ساختاری جا گرفتند و در هر مادۀ غذایی ساختار تغییر می‌کنه،‌ حتی اگه نسبت یا مقدار اجزاء تقریبا ثابت بمونه. آیا ساختار را در انتخاب‌ها یا توصیه‌های غذایی تاثیر می‌دیم؟
سیگنال سیری را دقیقا کی یا کیا صادر می‌کنند؟
آیا هر کاری اجدادمون می‌کردند ما هم باید بکنیم؟
آیا تلاش برای لاغر شدن هم‌جهت و هم‌راستا با تلاش‌ها برای زندگی سالم‌تره؟
آیا قوز کمر منِ لاغر [از جنس] همان مشکل چاقی توی چاق نیست؟
آیا هر گردی، گردوه؟
چرا توی مواد غذایی دست می‌بریم؟ چرا نان و برنج قهوه‌ای را سفید و پوست سیب‌زمینی را می‌کنیم؟ چرا تفالۀ سیب را از آبش جدا می‌کنیم؟
توزیع دستورالعمل‌ها، دستورالعمل‌های موسسات غذایی و دستورالعمل‌های رفرنس پزشکی و الخ، در جغرافیا به چه صورته؟
آیا کسب موفقیت‌های کوتاه‌مدت و گرفتن نتیجۀ لاغری از یک رژیم کاری به نتایجش در درازمدت هم داره؟ به راستی چرا می‌خوایم لاغر بشیم؟
آیا روزی می‌رسه؟ یا هر کس روزی خودش را فراهم می‌کنه؟!
هورمون سیری دیگه چیه؟
اندورفین کدام هورمونه؟
آیا دستگاه گوارش همۀ غذاها را به یه شکل پردازش می‌کنه؟
قبل از رژیم‌های کتوژنیک و مدیترانه‌ای و رژیم آمریکا و کشف آمریکا رژیم‌های بی‌نام و نشان چی بودند؟
آیا هیچ بار شده که همۀ رویکردهای تغذیه و سلامتی در سراسر جهان مقایسه و مطالعۀ تطبیقی بشند؟
آیا مرگ یک رویداده یا یک روند؟ سلامتی چی؟
چرا توی Dietary Guidelines for Americans, 2025–2030 دو بار اسم ترامپ اومده بود!؟
کارکرد یا هدف ما اجزاء انسانی در سیستم اجتماعیمون چیه؟ مگر تناسب اندام بدنسازی‌شده یا چاقی یا موارد مشابه نیست؟
آیا عطف نقیض دو گزارۀ «کم بخور و همیشه بخور» هم‌ارز همین ترکیب گزاره‌ها نیست؟ کدام ترکیب، ترکیب برنده است؟
آیا اگر برای رفع خستگی کارکرد زیرسیستم دهان از جهاز هاضمه را پایین بیاریم، مثلا طالبی و انبه را داخل مخلوط‌کن بریزیم و بنوشیم تا اینکه بجویم، طوریه؟!
اگه تفاله‌های هویج را توی آ‌ب هویج هم بزنیم و بخوریم، … ヾ(≧▽≦*)o
آیا شیمی بدن قضاوتش راجع به مرکبات و لیموترش، اسیدی است؟
شاخص گلیسمی چیه؟
چرا هر کی عاشق و دل‌باختۀ یه طعمیه؟ می‌تونیم چشم‌بسته بگیم عاشق‌های سینه‌چاک شیرینی بیشترند؟
چرا بعضی‌ها گیاه‌خوار میشند؟
چه کسانی، افراد چاق را گناهکار یا آلوده به گناه می‌خوانند؟

دارم یکی‌یکی مطالبی را که در حوزۀ سلامتی نوشته‌ای را می‌خونم. ترغیب شدم من هم سوالاتی را بپرسم. من هم سوال پرسیدن را دوست دارم. از همۀ این پرسش‌ها بعضی‌هاش در امتداد، بعضی در پاسخ و تعدادیشون همان‌ پرسش‌های تو با ادبیات و از منظر متفاوت هستند و بقیه هم شاید نسبتی نداشته باشند.

**Reply by pourmand1376** [reply]
سلام مسعود. سؤالات خوبی پرسیدی. 
جدا از بحث رژیم اهمیت سؤال رو کم‌کم دارم بیشتر بهش پی‌ می‌برم. خصوصاً تو دوره صوتی پیتر دراکر که شعبانعلی صحبت می‌کنه، یکی از تم‌های پررنگ اقای دراکر همین سؤال پرسیدن هست. این که سؤالات متفاوت می‌پرسه. با سؤالاتی که بقیه می‌پرسند قانع نمیشه. وقتی سؤالات متفاوت می‌پرسی، جواب‌های متفاوتی هم می‌گیری. 

کتاب پیتر عطیه هم در مسیر کمک به پاسخ دادن به این سؤالات می‌تونه کمک‌کننده باشه. موفق باشی.
